staging/src/k8s.io/client-go/testing/actions.go

```diff
@@ -74,8 +74,8 @@ func NewRootListAction(resource schema.GroupVersionResource, kind schema.GroupVe
 	action.Verb = "list"
 	action.Resource = resource
 	action.Kind = kind
-	labelSelector, fieldSelector, _ := ExtractFromListOptions(opts)
-	action.ListRestrictions = ListRestrictions{labelSelector, fieldSelector}
+	labelSelector, fieldSelector, resourceVersion := ExtractFromListOptions(opts)
```

Comment:
plumbing ResourceVersion seems ok, but we also have to plumb ResourceVersionMatch if we're going to start using ResourceVersion so the *meaning* of resourceVersion is honored